### ISSUE TITLE
prefer-primordials: Add check for getter properties

### ISSUE THREAD

**Issue by petamoriken** [denoland/deno_lint#1112]
```js
// call ArrayBuffer.prototype.byteLength getter function
new ArrayBuffer(10).byteLength;
```

should be changed to the following:

```js
ArrayBufferPrototypeGetByteLength(new ArrayBuffer(10));
```

---

Since type information is not available, it is the only way to check if the property names are defined as getter properties in ECMAScript:

`Symbol`

* `description`

`RegExp`

* `dotAll`
* `flags`
* `global`
* `hasIndices`
* `ignoreCase`
* `multiline`
* `source`
* `sticky`
* `unicode`

`TypedArray`

* `buffer`
* `byteLength`
* `byteOffset`
* `length` 👉 better to ignore it because it cause false positives for the `Array`s

`Map`, `Set`

* `size`

`ArrayBuffer`, `SharedArrayBuffer`

* `byteLength`

`DataView`

* `buffer`
* `byteLength`
* `byteOffset`
